Automattic/wp-calypso
# JP Manage Pricing: Technical polish - simplifying markup, moving CSS

## Proposed Changes

* This is cleanup PR - removing custom markup in FeaturedLicenseItemCard, FeaturedLicenseMultiItemCard, SimpleLicenseItemCard, and SimpleLicenseMultiItemCard.
* It also cleans up CSS - moving multichoice relevant CSS to the parent simple-item-card and featured-item-card CSS files as these are new features to those components and can be reused.
* It also fixes the location of the 'More Info' link in  FeaturedLicenseItemCard and FeaturedLicenseMultiItemCard, as these moved when removing custom markup.
* It also makes sure tracks events names are consistent.

## Testing Instructions

* To test, apply this PR.
* Visit `http://jetpack.cloud.localhost:3000/manage/pricing`. Confirm styling matches what is expected (see https://github.com/Automattic/wp-calypso/pull/87104 for the latest design expectations and conversations). Confirm behaviour matches what is expected as well (links open correct modals, with correct prices and quantities).

## Pre-merge Checklist
